annotate.el

```diff
@@ -210,18 +255,26 @@
         (message "Annotations saved."))))
 
 (defun annotate-actual-comment-start ()
+  "String for  comment start  related to current  buffer's major
+mode."
```

Comment:
Thank you for adding documentation!

Reply:
i hate writing documentation (like many others :-)) but i hate more code that is not documented!  ;-D